<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaltFilter</class>
 <widget class="QMainWindow" name="SaltFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MapEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="MyLabel" name="ImageShow">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>811</width>
      <height>571</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Procesar">
    <property name="geometry">
     <rect>
      <x>861</x>
      <y>137</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Procesar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1049</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>109</y>
      <width>107</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header location="global">mylabel.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
